code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -23,6 +23,11 @@
 	name = "[initial(name)] ([storedorgan.name])"
 
 /obj/item/autosurgeon/attack_self(mob/user)//when the object it used...
+	if(istype(storedorgan, /obj/item/organ/regenerative_core))
+		var/obj/item/organ/regenerative_core/cores = storedorgan
+		to_chat(user, span_notice("[src] fucking explodes!"))
```

Comment:
This needs better flavortext and should mention that it is exploding specifically because it is loaded with a legion core

Reply:
Updated

Reply:
Also updated the Suspicious Autosurgeon's description to better inform them that if they do this it's a bad idea.